### PR TITLE
postgresql_schema: Add drop_cascade option.

### LINKED ISSUE #101: postgresql drop schema
Hi there,
I am trying to destroy postgresql resource when schema contains depended resources.

terraform version v0.12.6

### Affected Resource(s)
Please list the resources as a list, for example:
- postgresql_schema

### Expected Behavior
Resource "postgresql_schema" has option "cascade" for deleting depended resources.

### Actual Behavior
Operation failed due to the fact that schema has depended resources. 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform destroy when schema has depended resources`

### PR DESCRIPTION
**Pull request by cyrilgdn**
This allows to delete cascade all dependent objects while deleting a schema.

fix #101

### CODE REVIEW

File: postgresql/resource_postgresql_schema_test.go [hunk 1]
```diff
@@ -196,6 +196,32 @@ func TestAccPostgresqlSchema_Database(t *testing.T) {
 	})
 }
 
+func TestAccPostgresqlSchema_DropCascade(t *testing.T) {
+	var testAccPostgresqlSchemaConfig = `
+resource "postgresql_schema" "test_restrict" {
```
[the remark this answers is not in the document]
Comment (cyrilgdn) [reply postgresql/resource_postgresql_schema_test.go]: Indeed, I changed the test behavior in the middle but not the resource name !

[the remark this answers is not in the document]
Comment (cyrilgdn) [reply postgresql/resource_postgresql_schema_test.go]: Updated

**Comment by navi86** [comment]
@cyrilgdn 
I've tried to add test for validation that it's not possible to drop schema when it has dependent resource, but I couldn't find solution, because "expectError" works for test steps and there is no way to add it for destroy step. Am I right ?

**Comment by cyrilgdn** [comment]
> @cyrilgdn
> I've tried to add test for validation that it's not possible to drop schema when it has dependent resource, but I couldn't find solution, because "expectError" works for test steps and there is no way to add it for destroy step. Am I right ?

AFAIK yes, I also tried to create this test (that's why the naming was not coherent) and tried this `expectError` too but didn't managed to do it.
I'll ask on the Terraform community Slack if there's a way. But for now I just rely on the fact that this test fails if I remove `drop_cascade`

**Comment by cyrilgdn** [comment]
@navi86 Are you agree to merge it like that? I want to create a new release (with this feature inside) and Hashicorp devs will be OOO for the end of the year so it'd be great if I can ask them today.

**Comment by navi86** [comment]
> @navi86 Are you agree to merge it like that? I want to create a new release (with this feature inside) and Hashicorp devs will be OOO for the end of the year so it'd be great if I can ask them today.

sure, LGTM

**Pull request merged**